tapdb/assets_store_test.go

```diff
@@ -2088,7 +2090,29 @@ func TestTransferOutputProofDeliveryStatus(t *testing.T) {
 
 	// At this point, we should be able to query for the log parcel, by
 	// looking for all unconfirmed transfers.
-	assetTransfers, err := db.QueryAssetTransfers(ctx, TransferQuery{})
+	assetTransfers, err := db.QueryAssetTransfers(ctx, TransferQuery{
```

Comment:
Didn't spot this the first time; I think we want to add assertions on the new `AnchorTxBlockHash` field for the asset transfers here?

IIUC before `ConfirmChainAnchorTx`, it should be len == 0. And therefore we should see the same transfer if  `PendingTransfersOnly` was true.

After the TX confirmation the block hash should match `randBlockHash`.